### ISSUE TITLE
App Crashes when closing composer window in fullscreen (osX Sierra)

### ISSUE THREAD

**Issue by mirisuzanne** [nylas/N1#2792]
N1 Version: 0.4.49-c9c04b7 (0.4.49-c9c04b7)
OS Version: 10.12 Beta (16A313a)

When the app is in fullscreen mode, new messages appear in a tab. I love that UX, but any time that window is closed (either by sending the message, or clicking the close icon), the entire application crashes.

I know the beta OS may be causing this behavior, but thought you should know about it. There are no plugins installed.

**Comment by emorikawa** [comment]
@mirisuzanne Yes. We're able to reproduce this on Sierra. This is definitely a problem with Electron and the Sierra full screen window management system. We need to work on getting this fixed upstream.

**Comment by TajviaWillis** [comment]
+1

**Comment by samgabell** [comment]
This is an immensely painful issue, is there any timeline on a fix?

**Comment by retrorism** [comment]
Same problem here with the non-beta of Sierra. Good to know that you're looking at it, looking forward to the fix (Window management in Atom does not seem to be affected by this issue, by the way).

**Comment by WesGBrooks** [comment]
Noticing this same issue on non-beta Sierra as well (10.12 16A323), Version 0.4.56-816edde of Nylas.

**Comment by BridgetP** [comment]
+1

**Comment by dbenic** [comment]
yes, it's crashing.
Mac: 10.12 
Nylas: Version 0.4.56-816edde (0.4.56-816edde)

**Comment by jstejada** [comment]
another incident: https://nylas.zendesk.com/agent/tickets/6656
